assets/csidriver.yaml

```diff
@@ -5,6 +5,8 @@ metadata:
   annotations:
     # This CSIDriver is managed by an OCP CSI operator
     csi.openshift.io/managed: "true"
+  labels:
+    security.openshift.io/csi-ephemeral-volume-profile: "restricted"
```

Comment:
so to follow up on our conversation this morning @coreydaley @adambkaplan @siamaksade @rgormley the storage team already set the label on our driver that per https://github.com/openshift/jenkins-sync-plugin/blob/master/hack/tag-ci-image.sh means anybody who is a `restricted` user or higher can use the shared resource driver.